update-center-source.properties

```diff
@@ -1,12 +1,20 @@
 plugins=aemrules,ansible,apigee,authaad,authbitbucket,authgoogleoauth,authoidc,checkstyle,citymodel,cloudformation,clover,cnesreport,cobertura,codehawk,communitybsl,communityrust,crowd,dependencycheck,detekt,devon4j,findbugs,fxcop,groovy,hadolint,ibmixaemrules,jdepend,jqassistant,lits,l10nde,l10nfr,l10nel,l10nes,l10nit,l10nja,l10nko,l10nru,l10nzh,l10nzhtw,mulesoft,mutationanalysis,mybatis,openedge,pmd,quboo,shellcheck,rci,scmcvs,scmjazzrtc,scmtfvc,scmmercurial,smells,softvis3d,sonargraphintegration,webdriver,xanitizer,yaml
 scanners=scannercli,scannermsbuild,scannerant,scannermaven,scannergradle,scannerazure,scannerjenkins
 
-devVersion=9.5-SNAPSHOT
-publicVersions=8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4
+devVersion=9.6-SNAPSHOT
```

Comment:
Do we even still use this?

Reply:
Good question 🤷‍♂️
@pierre-guillot-sonarsource di you know?

Reply:
I don't remember. Maybe @aurelien-poscia-sonarsource you do remember?

Reply:
It is still used by the update-center to force the re-download of SQ zip before generating the html matrix; to make sure we use the latest code in case we are referencing an artifact under development.